grpclog/internal/loggerv2_test.go

```diff
@@ -20,11 +20,86 @@ package internal
 
 import (
 	"bytes"
+	"encoding/json"
 	"fmt"
+	"io"
+	"os"
+	"os/exec"
 	"regexp"
+	"strings"
 	"testing"
 )
 
+// check if b is in the format of:
+//
+//	2017/04/07 14:55:42 WARNING: WARNING
```

Comment:
fix indenentation

Reply:
I didn't add this, I just moved it to put helper functions at the top, like you requested previously.

The indentation here is being done by `gofmt`, if I remove it `gofmt` adds it back. Did you still want me to remove i?t